recipes/llvmdev/meta.yaml

```diff
@@ -0,0 +1,51 @@
+# (Mostly) blatant copy of https://github.com/numba/llvmlite/tree/master/conda-recipes/llvmdev
+
+{% set version = "3.8.1" %}
+{% set sha256 = "6e82ce4adb54ff3afc18053d6981b6aed1406751b8742582ed50f04b5ab475f9" %}
+
+package:
+  name: llvmdev
+  version: {{ version }}
+
+source:
+  fn: llvm-{{ version }}.src.tar.xz
+  url: http://llvm.org/releases/{{ version }}/llvm-{{ version }}.src.tar.xz
+  sha256: {{ sha256 }}
+  patches:
+    # https://llvm.org/bugs/show_bug.cgi?id=15356
+    - llvm-elf-relocation.patch
+    # https://llvm.org/bugs/show_bug.cgi?id=25788
+    - llvm-parse-fp-constants.patch
+    # https://llvm.org/bugs/show_bug.cgi?id=25565
+    - llvm-build-race-condition.patch
+    # http://lists.llvm.org/pipermail/llvm-dev/2016-January/094520.html
+    - llvm-lto-static.patch  # [win]
+
+build:
+  number: 6
+
+requirements:
+  build:
+    - cmake
+    # Needed to unpack the source tarball
+    - m2-xz  # [py27 and win32]
+    - m2w64-xz  # [py27 and win64]
+    # Needed to build LLVM
+    - python
+    - system  # [linux and not armv7l]
+  run:
+    - system  # [linux and not armv7l]
+
+test:
+  commands:
+    - llvm-config --libs  # [not win]
+    - llc -version
+
+about:
+  home: http://llvm.org/
+  license: Open Source (http://llvm.org/releases/{{ version }}/LICENSE.TXT)
```

Comment:
You can use `NCSA` (see https://opensource.org/licenses/NCSA) and remove the URL from here. If you want you can add the `LICENSE.TXT` in the `license_file` field right below this line.